openwisp_monitoring/monitoring/apps.py

```diff
@@ -12,5 +15,15 @@ class MonitoringConfig(AppConfig):
 
     def ready(self):
         # create influxdb database if doesn't exist yet
-        create_database()
+        for n in range(5):
+            try:
+                create_database()
+                break
+            except InfluxDBClientError:
+                self.warn_and_delay(n)
         setattr(settings, 'OPENWISP_ADMIN_SHOW_USERLINKS_BLOCK', True)
+
+    def warn_and_delay(self, n):
+        print('Got error while connecting to timeseries DB. '
+              f'Retrying again in 3 seconds. Attempt {n} out of 5')
```

Comment:
```python
>>> list(range(5))
[0, 1, 2, 3, 4]
```


At the first attempt this will be returned as output:

`Retrying again in 3 seconds. Attempt 0 out of 5`

Add 1 to n to solve this. Please also rename `n` to `attempt_number`